BUILD

```
@@ -216,6 +216,23 @@ go_sdk_tool(
     goroot_relative_path = "bin/gofmt",
 )
 
+alias(
+    name = "goimports",
+    actual = "@org_golang_x_tools//cmd/goimports",
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "goimports_bin",
+    outs = ["goimports.sh"],
+    cmd = """
+echo '#!/bin/bash' > $@
+echo 'exec $(location @org_golang_x_tools//cmd/goimports) "$$@"' >> $@
```

Slightly easier to read and also emitting the runfiles path rather than the execpath to `goimports`, which will be necessary with Bazel 8 and `--nolegacy_external_runfiles`:
```suggestion
echo <<EOF > $@
#!/bin/bash
exec $(rootpath @org_golang_x_tools//cmd/goimports) "$$@"
EOF
```